download_progress.py

```diff
@@ -72,13 +73,13 @@ def add_download(self, url, filename):
 
         # URL and Filename
         label_sizer = wx.BoxSizer(wx.HORIZONTAL)
-        url_label = wx.StaticText(item_panel, label=f"Downloading: {filename}")
+        url_label = wx.StaticText(item_panel, label=_("Downloading: {filename}"))

```

Comment:
This doesn't work.
You need something like this.
url_label = wx.StaticText(item_panel, label=_("Downloading: %s") % filename)